bazel/repositories.bzl

```diff
@@ -212,3 +197,66 @@ def envoy_dependencies(path = "@envoy_deps//", skip_protobuf_bzl = False, skip_t
     python_deps(skip_targets)
     cc_deps(skip_targets)
     envoy_api_deps(skip_targets)
+
+def com_github_fmtlib_fmt():
+  native.new_http_archive(
+      name = "com_github_fmtlib_fmt",
+      urls = [
+          "https://github.com/fmtlib/fmt/releases/download/4.0.0/fmt-4.0.0.zip",
+      ],
+      sha256 = "10a9f184d4d66f135093a08396d3b0a0ebe8d97b79f8b3ddb8559f75fe4fcbc3",
+      strip_prefix = "fmt-4.0.0",
+      build_file = "//bazel/external:fmtlib.BUILD",
```

Comment:
Does this work when the repo is consumed by other `git_repository` or `local_repository`? Make sure this works in envoy-filter-example, or istio/proxy.

Reply:
`build_file` takes a label, resolved relative to the current repository (https://github.com/bazelbuild/bazel/issues/682). This will work when building Envoy as part of a larger workspace.

I believe older versions of Bazel (< 4.something?) used plain paths for `build_file`. If we need to support those versions, I can change to `build_file_content`. But I'd prefer not to, because it's ugly.

Reply:
No I don't need older version support, Good to know it works now.